src/main/java/geraldbot/Launcher.java

```diff
@@ -0,0 +1,12 @@
+package geraldbot;
+
+import javafx.application.Application;
+
+/**
+ * A launcher class to workaround classpath issues.
+ */
```

Comment:
Nothing wrong with this javaDocs as well. Personally, using Constructs a launcher class ... sounds nicer